integration_tests/src/janus.rs

```diff
@@ -189,7 +189,7 @@ impl JanusInProcess {
             min_aggregation_job_size: 1,
             max_aggregation_job_size: 100,
             aggregation_job_creation_report_window: 5000,
-            late_report_grace_period_s: 3600,
+            late_report_grace_period_units: 10,
```

Comment:
Why 10? Though I don't think the 3600 we had previously was particularly carefully considered.

Reply:
in dd359ca08df24fbbfabe59bb509f892e646aab53